src/main/java/skeltal/Skeltal.java

```diff
@@ -0,0 +1,29 @@
+package skeltal;
+
+import java.util.Scanner;
+
+/**
+ * Represents the main class of the Skeltal chatbot app.
+ */
+public class Skeltal {
+
+    /**
+     * A method that scans for user input and prints a response.
+     */
+    public static void run() {
+        TaskList.loadTaskList(Storage.loadFile());
+        Ui.introduction();
+        Scanner sc = new Scanner(System.in);
+        boolean shutdown = false;
```

Comment:
Perhaps this can be renamed to sound more like a boolean? According to the [naming guidelines](https://se-education.org/guides/conventions/java/intermediate.html#naming)